packages/cli/src/commands/setup/jobs/jobsHandler.js

```diff
@@ -0,0 +1,140 @@
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+
+import { getDMMF } from '@prisma/internals'
+import chalk from 'chalk'
+import execa from 'execa'
+import { Listr } from 'listr2'
+
+import { getPaths, transformTSToJS, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+import { isTypeScriptProject } from '../../../lib/project'
+
+const MODEL_SCHEMA = `
+model BackgroundJob {
+  id        Int       @id @default(autoincrement())
+  attempts  Int       @default(0)
+  handler   String
+  queue     String
+  priority  Int
+  runAt     DateTime?
+  lockedAt  DateTime?
+  lockedBy  String?
+  lastError String?
+  failedAt  DateTime?
+  createdAt DateTime  @default(now())
+  updatedAt DateTime  @updatedAt
+}
+`
+
+const getModelNames = async () => {
+  const schema = await getDMMF({ datamodelPath: getPaths().api.dbSchema })
+
+  return schema.datamodel.models.map((model) => model.name)
+}
+
+const addModel = () => {
+  const schema = fs.readFileSync(getPaths().api.dbSchema, 'utf-8')
+
+  const schemaWithUser = schema + MODEL_SCHEMA
+
+  fs.writeFileSync(getPaths().api.dbSchema, schemaWithUser)
+}
+
+const tasks = async ({ force }) => {
+  const modelExists = (await getModelNames()).includes('BackgroundJob')
+
+  return new Listr(
+    [
+      {
+        title: 'Creating job model...',
+        task: () => {
+          addModel()
+        },
+        enabled: () => !modelExists,
```

Comment:
If setup is run with --force I'd expect this step to also run

Reply:
There is a 0% likelihood you want a second copy of the same model in your schema (which will throw errors when Prisma finds out) so I decided to skip if the model is already found. 

I don’t think anyone will ever run the setup again, but I kept —force in there to be consistent with the others.

I’d either keep it like is or remove the —force option altogether and just have text output explaining why it couldn’t complete.

Reply:
Of course not a second model 😄 I'd expect it to overwrite, just like --force does for files

Reply:
So I borrowed the logic for how to add a model to the schema file from the user one you did for dbAuth, but I see you aren't doing any kind of de-duping, it just adds the model again if you `--force`—I definitely don't think we should do that! We could do a regex find/replace, but if you changed the model name it wouldn't find the existing one and so add a second one anyway. I imagine a codemod has the same drawbacks as you have to trigger off of *something*, but the user could have changed it.

Very few, if anyone, is going to run a setup command again in the same project, I don't think it's worth the complexity, especially considering it's far from foolproof.